ext/rugged/rugged_revwalk.c

```diff
@@ -192,6 +192,35 @@ static VALUE rb_git_walker_simplify_first_parent(VALUE self)
 
 /*
  *  call-seq:
+ *    walker.count -> Fixnum
+ *
+ *  Returns the amount of objects a walker iterated over.
+ */
+static VALUE rb_git_walker_count(VALUE self)
+{
+    git_revwalk *walk;
+    git_oid commit_oid;
+    int error = 0;
+    long count = 0;
```

Comment:
I left it as `long` since I'm not 100% sure if `NUM2INT` can deal with `uint64_t` but I'll give it a try.